src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -2047,15 +2049,15 @@ public function prepare_item_for_response( $item, $request ) {
 	}
 
 	/**
-	 * Overwrites the default protected title format.
+	 * Overwrites the default protected and private title format.
 	 *
-	 * By default, WordPress will show password protected posts with a title of
-	 * "Protected: %s", as the REST API communicates the protected status of a post
-	 * in a machine-readable format, we remove the "Protected: " prefix.
+	 * By default, WordPress will show password protected or private posts with a title of
+	 * "Protected: %s" or "Private: %s", as the REST API communicates the status of a post
+	 * in a machine-readable format, we remove the prefix.
 	 *
 	 * @since 4.7.0
```

Comment:
Sorry to be pedantic, but does this change warrant a `@since` annotation?

Reply:
For me it doesn't, the function didn't change.

Reply:
All good! 🚀